pygfunction/media.py

```diff
@@ -0,0 +1,189 @@
+from __future__ import absolute_import, division, print_function
+
+from CoolProp.CoolProp import PropsSI
+import warnings
+
+
+class Fluid:
+    """
+        An object for handling the fluid properties
+
+        Parameters
+        ----------
+        mixer: str
+            The mixer for this application should be one of:
+                - 'MEG' - Ethylene glycol mixed with water
+                - 'MPG' - Propylene glycol mixed with water
+                - 'MEA' - Ethanol mixed with water
+                - 'MMA' - Methanol mixed with water
+        percent: float
+            Mass fraction of the mixing fluid added to water (in %).
+            Lower bound = 0. Upper bound is dependent on the mixture.
+        T: float, optional
+            The temperature of the fluid (in Celcius).
+            Default is 20 degC.
+        P: float, optional
+            The pressure of the fluid (in Pa).
+            Default is 101325 Pa.
+
+        Examples
+        ----------
+        >>> import pygfunction as gt
+        >>> T = 20.     # Temp at 20 C
+        >>> gage_P = 20  # PsiG
+        >>> atm_P = 14.69595
+        >>> P = (gage_P + atm_P) * 6894.75728  # Pressure in Pa
+
+        >>> # pure water
+        >>> fluid = gt.media.Fluid(T=T, P=P)
+        >>> print(fluid)
```

Comment:
This `pure water` example does not provide a `mixer` and `percent` variable. It appears the code would break now that `mixer` and `percent` are required.

Reply:
I do not think the Melinder fluids should be used for water. The properties do not match at `percent=0.`:
```
import pygfunction as gt
print(gt.media.Fluid(T=20, mixer='MEG', percent=0.).rho)
print(gt.media.Fluid(T=20, mixer='MPG', percent=0.).rho)
print(gt.media.Fluid(T=20, mixer='MEA', percent=0.).rho)
print(gt.media.Fluid(T=20, mixer='MMA', percent=0.).rho)
```
returns:
```
997.3492877107519
998.5158915716694
998.1780533839934
998.4145842238611
```

A dedicated `Water` class could resolve the issue.